src/main.rs

```diff
@@ -118,11 +118,13 @@ fn list() {
         println!("No existing workspaces.\nRun `workspace new <NAME>` to create one.");
         return;
     }
-
+    
+    let l = (*all).iter().max_by(|ws1, ws2| ws1.name.len().cmp(&mut ws2.name.len())).unwrap().name.len();
```

Comment:
The way this is done would be optimal if we needed the workspace with the longest name. Here however, we only want to know the length of the longest name, so we can `map` to name lengths and `fold` with `std::cmp::max`. I'll push a commit with this change and `rustfmt`ing. Also, use descriptive variable names! Otherwise, nice work on the alignments.